include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -1222,71 +1226,53 @@ namespace internal
         /**
          * Return vertex DoF indices.
          */
-        template <int structdim, typename DoFProcessor, bool level_dof_access>
+        template <typename DoFProcessor>
         DEAL_II_ALWAYS_INLINE void
-        process_vertex_dofs(
-          const dealii::DoFAccessor<structdim, dim, spacedim, level_dof_access>
-            &                       accessor,
-          const unsigned int        vertex,
-          types::global_dof_index *&dof_indices_ptr,
-          const unsigned int        fe_index,
-          const DoFProcessor &      dof_processor) const
+        process_vertex_dofs(DoFHandler<dim, spacedim> &dof_handler,
+                            const unsigned int         vertex_index,
+                            const unsigned int,
+                            types::global_dof_index *&dof_indices_ptr,
+                            const DoFProcessor &      dof_processor) const
         {
           const unsigned int n_indices =
-            accessor.get_fe(fe_index).template n_dofs_per_object<0>();
+            dof_handler.get_fe(0).template n_dofs_per_object<0>();
```

Comment:
What happened with `fe_index`?

Reply:
OK. This is the MG path. Wouldn't it make sense to add an assert that `fe_index` is ineed 0 or default?

Reply:
Yes, that makes sense.

Reply:
Note that I added the check in line 1344 below, not here, as I do not want to pick up the index here again.